### PR TITLE
Publish pullsheet leaderboard to website

### LINKED ISSUE #11445: pullsheet leaderborad page in contributors website section

### PR DESCRIPTION
**Pull request by andriyDev**
fixes #11445.

Now we have an automated way to generate contribution charts for all our releases!

[Screenshot](https://user-images.githubusercontent.com/22285953/118892383-af9faa00-b8b5-11eb-905e-aba1417b2eff.png)

### CODE REVIEW

**Comment by andriyDev** [comment]
Should we perhaps skip minor releases (e.g. 1.18.1)?

**Review by medyagh — CHANGES_REQUESTED** [review]
this is really cool and fun to look at thank you for this PR
Lets move this under `Contributing`
under a new section called "triage & contribution leaderboard"
Screen Shot 2021-05-19 at 3 22 53 PM


if it is easy lets add all the releases of minikube. (if it can be done through a VM that wont take ur computer time)

if you decide to skip minor releases, please make sure the releases are exactly between major releases
like minikube v1.17.0  v1.18.0
and then 1.18.0 - v1.19.0

but I think for sake of simplicity we should just do all releases regardless of minor or major
but ensure to publish all

**Comment by medyagh** [comment]
@andriyDev lets also add the Date of each release into the list,
minikube Version - Date ...

**Comment by andriyDev** [comment]
@medyagh Done! Added date to each release title. Published all releases. I just went and rewrote the commit history.

File: hack/release_notes.sh [hunk 1]
```diff
@@ -64,3 +64,6 @@ echo "Thank you to our triage members for this release!"
 echo ""
 AWK_ISSUE_COMMENTS='NR>1{arr[$4] += $7}END{for (a in arr) printf "%d %s\n", arr[a], a}'
 "${DIR}/pullsheet" issue-comments --since "$recent_date" --repos kubernetes/minikube --token-path $DIR/gh_token.txt --logtostderr=false --stderrthreshold=2 | awk -F ',' "$AWK_ISSUE_COMMENTS" | sort -k1nr -k2d  | awk -F ' ' "$AWK_FORMAT_ITEM" | head -n 5
+
+echo ""
+echo "Don't forget to run \"hack/update_contributions.sh\"!"
```
Comment (medyagh) [inline hack/release_notes.sh:69]: lets provide a "make target " for example "make update-leaderboard"

**Review by medyagh — CHANGES_REQUESTED** [review]

File: hack/release_notes.sh (hunk 1 above)
Comment (andriyDev) [reply hack/release_notes.sh:69]: Done.

**Review by medyagh — APPROVED** [review]

**Pull request merged**